README.md

```diff
@@ -12,14 +12,14 @@ The `dbt-dremio` package contains all of the code enabling dbt to work with [Dre
 
 The dbt-dremio package supports both Dremio Cloud and Dremio Software (versions 22.0 and later).
 
-Installing the dbt-dremio package will install or update dbt-core to version 1.3.2.
+Installing the dbt-dremio package will install dbt-core to version 1.3.2 if there is no existing dbt-core installation or if the existing dbt-core installation is older than 1.2.0. If the installed dbt-core has a version >=1.2 and <=1.3.2, the existing installation will be used and not upgraded to v1.3.2.
 
 > Prior to version 1.1.0b, dbt-dremio was created and maintained by [Fabrice Etanchaud](https://github.com/fabrice-etanchaud) on [their GitHub repo](https://github.com/fabrice-etanchaud/dbt-dremio). Code for using Dremio REST APIs was originally authored by [Ryan Murray](https://github.com/rymurr). Contributors in this repo are credited for laying the groundwork and maintaining the adapter till version 1.0.6.5. The dbt-dremio adapter is maintained and distributed by Dremio starting with version 1.1.0b.
 
 ## Getting started
 
 -   [Install dbt](https://docs.getdbt.com/docs/installation)
-    -   Note that dbt-dremio requires dbt-core 1.3.2.
+    -   Note that dbt-dremio requires dbt-core >=1.2.0 and <=1.3.2.
```

Comment:
I know that his content came from the other adapters' README, but it is a bit silly as it is not completely true.

Reply:
Let's change line 22 to:

Install dbt-dremio (link to our documentation on dbt site)
- Version 1.3.2 of dbt-dremio requires dbt-core >= 1.2.0 and <=1.3.2. Installing dbt-dremio will automatically upgrade existing dbt-core versions earlier than 1.2.0 to 1.3.2, or install dbt-core v1.3.2 is no version of dbt-core is found.

Feel free to wordsmith the above.